paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/query/RemoteTableQuery.java

```diff
@@ -109,4 +111,9 @@ public InternalRowSerializer createValueSerializer() {
     public void close() throws IOException {
         client.shutdown();
```

Comment:
`networkClient.shutdown().get(10L, TimeUnit.SECONDS);`

10 s is not enough?

Reply:
The two CI logs mentioned in issue #3641 show that the running time is greater than 10s.

```
https://github.com/apache/paimon/actions/runs/9729518905/job/26851511142
https://github.com/apache/paimon/actions/runs/9729521981/job/26851517274
```

![image](https://github.com/apache/paimon/assets/37063904/6fe278e8-a2c5-4a09-b803-09258c8477a5)

Reply:
can we just modify this to 60 s?

Reply:
keep your changes and modify this to 60 s.

Reply:
Done.